src/PrestaShopBundle/Form/Admin/Sell/Product/OptionsType.php

```diff
@@ -0,0 +1,185 @@
+                'required' => false,
+                'label' => $this->trans('UPC barcode', 'Admin.Catalog.Feature'),
+                'constraints' => [
+                    //@todo: adjust TypedRegex use UPC VO
+                    new Regex('/^[0-9]{0,12}$/'),
+                ],
+                'empty_data' => '',
+            ])
+            ->add('ean13', TextType::class, [
+                'required' => false,
+                'error_bubbling' => true,
+                'label' => $this->trans('EAN-13 or JAN barcode', 'Admin.Catalog.Feature'),
+                'constraints' => [
+                    //@todo: adjust TypedRegex
+                    new Regex('/^[0-9]{0,13}$/'),
+                ],
+                'empty_data' => '',
+            ])
+            ->add('isbn', TextType::class, [
+                'required' => false,
+                'label' => $this->trans('ISBN', 'Admin.Catalog.Feature'),
+                'constraints' => [
+                    //@todo: adjust TypedRegex
+                    new Regex('/^[0-9-]{0,32}$/'),
+                ],
+                'empty_data' => '',
+            ])
+            ->add('reference', TextType::class, [
+                'required' => false,
+                'label' => $this->trans('Reference', 'Admin.Global'),
+                'empty_data' => '',
+            ])
+            ->add('show_condition', CheckboxType::class, [
+                'required' => false,
+                'label' => $this->trans('Display condition on product page', 'Admin.Catalog.Feature'),
+            ])
+            ->add('condition', ChoiceType::class, [
+                'choices' => [
+                    $this->trans('New', 'Shop.Theme.Catalog') => 'new',
+                    $this->trans('Used', 'Shop.Theme.Catalog') => 'used',
+                    $this->trans('Refurbished', 'Shop.Theme.Catalog') => 'refurbished',
+                ],
+                'attr' => [
+                    'class' => 'custom-select',
+                ],
+                'required' => true,
+                'label' => $this->trans('Condition', 'Admin.Catalog.Feature'),
+            ])
+//            ->add('suppliers', ChoiceType::class, [
+//                'choices' => $this->suppliers,
+//                'expanded' => true,
+//                'multiple' => true,
+//                'required' => false,
+//                'attr' => [
+//                    'class' => 'custom-select',
+//                ],
+//                'label' => $this->trans('Suppliers', 'Admin.Global'),
+//            ])
+//            ->add('default_supplier', ChoiceType::class, [
+//                'choices' => $this->suppliers,
+//                'expanded' => true,
+//                'multiple' => false,
+//                'required' => true,
+//                'attr' => [
+//                    'class' => 'custom-select',
+//                ],
+//                'label' => $this->trans('Default suppliers', 'Admin.Catalog.Feature'),
+//            ])
```

Comment:
Hi, What about these lines?

Reply:
don't rush to review, its still WIP :wink: